### PR TITLE
V1.6.2

### PR DESCRIPTION
**Pull request by esaari**
Branched off `master` - supersedes #22

https://github.com/webdriverio-community/wdio-rerun-service/pull/20 + backport specific changes
- adds unit tests

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -125,7 +130,7 @@ const RerunService = require('wdio-rerun-service');
 export.config = {
     // ...
     services: [
-        [RerunService, {
```
Comment (mikesalvia) [inline README.md:128]: This is incorrect

File: README.md [hunk 2]
```diff
@@ -125,7 +130,7 @@ const RerunService = require('wdio-rerun-service');
...
+        [RerunService], {
             ignoredTags: ['@known_bug']
         }]
```
Comment (mikesalvia) [inline README.md:135]: ending array item

Comment (esaari) [reply README.md]: should be fixed

File: README.md (hunk 1 above)
Comment (esaari) [reply README.md]: Take another look

**Pull request merged**